tests/components/unifi/test_switch.py

```diff
@@ -368,6 +392,74 @@ async def test_remove_switches(hass):
     assert block_switch is None
 
 
+async def test_block_switches(hass):
+    """Test the update_items function with some clients."""
+    controller = await setup_unifi_integration(
+        hass,
+        options={
+            CONF_BLOCK_CLIENT: [BLOCKED["mac"], UNBLOCKED["mac"]],
+            CONF_TRACK_CLIENTS: False,
+            CONF_TRACK_DEVICES: False,
+        },
+        clients_response=[UNBLOCKED],
+        clients_all_response=[BLOCKED],
+    )
+
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 2
+
+    blocked = hass.states.get("switch.block_client_1")
+    assert blocked is not None
+    assert blocked.state == "off"
+
+    unblocked = hass.states.get("switch.block_client_2")
+    assert unblocked is not None
+    assert unblocked.state == "on"
+
+    controller.api.websocket._data = {
+        "meta": {"message": MESSAGE_EVENT},
+        "data": [EVENT_BLOCKED_CLIENT_UNBLOCKED],
+    }
+    controller.api.session_handler(SIGNAL_DATA)
+    await hass.async_block_till_done()
+
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 2
+    blocked = hass.states.get("switch.block_client_1")
+    assert blocked is not None
+    assert blocked.state == "on"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)